src/components/resources/CreateResources.js

```diff
@@ -0,0 +1,285 @@
+                    right: '20px',
+                    zIndex: 9999
+                }}
+            />
+        );
+    });
+
+    const resetForm = () => {
+        setResourcesTitle('');
+        setResourcesLink('');
+    }
+
+    const validateResources = () => {
+
+        // console.log(" enableError is currently: " + enableError)
+
+        if (resourcesTitle === '') {
+            setResourcesTitleError(true);
+            setResourcesTitleErrorText('Please enter your resource title');
+            return false;
+        } else if (resourcesLink === '') {
+            setResourcesLinkError(true);
+            setResourcesLinkErrorText('Please enter the link to your resource');
+            return false;
+        } else {
+
+            setOpen(true);
+
+            var newResources = {
+                resourcesTitle: resourcesTitle,
+                resourcesLink: resourcesLink
+            }
+
+            // console.log(format(eventDate))
+            loadCreateResources();
+            resetForm();
+            setsuccessfullSubmissionMsg(true);
+            setTimeout(() => {
+                setsuccessfullSubmissionMsg(false);
+            }, 3000);
+
+            return false;
+        }
+    };
+
+    const useStyles = makeStyles((theme) => ({
+        formControl: {
+            margin: theme.spacing(1),
+            minWidth: 120,
+        },
+        selectEmpty: {
+            marginTop: theme.spacing(2),
+        },
+        root: {
+            '& > *': {
+                margin: theme.spacing(1),
+                width: '50ch',
+            },
+        },
+        paper: {
+            padding: theme.spacing(2),
+            textAlign: 'center',
+        },
+    }));
+
+    const classes = useStyles();
+
+    const loadCreateResources = () => {
+        callApiCreateResources()
+    }
+
+    const callApiCreateResources = async () => {
+
+        const url = `${REACT_APP_API_ENDPOINT}/createResources`;
+        console.log(url);
+        //console.log(eventDate);
+
+        try {
+            const response = await fetch(url, {
+                method: 'POST',
+                headers: {
+                    'Content-Type': 'application/json'
+                },
+                body: JSON.stringify({
+                    creatorID: userID,
+                    resourcesTitle: resourcesTitle,
+                    resourcesLink: resourcesLink
+                })
+            });
+
+            //console.log("got past the const response thing");
+            if (!response.ok) {
+                throw new Error(`${response.status}: ${response.statusText}`);
+            }
+
+            const data = await response.json();
+            console.log(data);
+        } catch (error) {
+            console.error(error);
+        }
+    }
+
+    return (
+        <div>
+            <NavigationBar></NavigationBar>
+
+            <Typography variant="h4" color="inherit" component="div" noWrap>
+                Create Resources
```

Comment:
Maybe it should be named to Submit a Resource? Not really sure!

Reply:
Yeah I agree